src/auto-evo/EvolutionaryTree.cs

```diff
@@ -136,6 +136,13 @@ public class EvolutionaryTree : Control
     [Signal]
     public delegate void SpeciesSelected(int generation, uint id);
 
+    /// <summary>
+    ///   Make use of tree-generated data to help exporting.
```

Comment:
```suggestion
    ///   Allows access to tree-generated data. This helps with data exporting, for example.
```
Would that be clearer interpretation of what this comment is saying?